### DISCUSSION TITLE
DB 설계 관련 정보들을 git으로 함께 관리

### DISCUSSION THREAD

**Discussion by jlee38266** [OmokNoonE/OnionHotSayYo-backend#30] [생각 & 제안]
DB 모델링에 사용되었던 DA# 파일과 테스트를 위해서 사용한 DDL 및 더미데이터를 git으로 관리하면 좋지 않을까 의견 제시합니다

**Comment by CJC0512** [comment]
동의합니다. 별도의 Repsitory로 빼서 관리하는 것도 나쁘지 않다고 생각합니다.
